README.md

```diff
@@ -96,12 +94,7 @@ This method dials a transport to the Peer listening on `multiaddr`.
 
 `callback` should follow the `function (err, conn)` signature.
 
-`conn` is the same `conn` that gets returned by call, which should follow [`interface-connection`](https://github.com/diasdavid/interface-connection). This `conn` object can emit 3 extra events:
```

Comment:
I would like to keep it there too, it is convenient when used with waterfall flows